src/core/core.helpers.js

```diff
@@ -466,14 +466,20 @@ module.exports = function(Chart) {
 	helpers.getConstraintHeight = function(domNode) {
 		return getConstraintDimension(domNode, 'max-height', 'clientHeight');
 	};
+	helpers.calculatePadding = function(container, padding, parentDimension) {
```

Comment:
We don't want this method to be part of the public API.

Can you prefix the method with `_` and add a private comment:

```javascript
/**
 * @private
 */
helpers._calculatePadding = function(...
```